runners/sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala

```diff
@@ -16,8 +16,27 @@ class SbtMutantRunner(state: State, processRunner: ProcessRunner, sourceCollecto
     implicit config: Config)
     extends MutantRunner(processRunner, sourceCollector) {
 
+  private lazy val filteredSystemProperties = {
+    // Matches strings that start with one of the options between brackets
+    val regex = "^(java|sun|file|user|jna|os|sbt|jline|awt|user).*"
+
+    val filteredProps =
+      sys.props.toList
+        .filterNot { case (key, _) => key.matches(regex) }
+        .map { case (key, value) => s"-D$key=$value" }
+
+    if (filteredProps.nonEmpty) {
+      debug("System properties added to the forked JVM")
```

Comment:
Probably cleaner to put this in one debug statement, something like `debug("System properties added to the forked JVM: ${filteredProps.mkString(", ")}")